Public/Src/Cache/ContentStore/Library/Utils/BandwidthChecker.cs

```diff
@@ -128,5 +132,60 @@ async Task impl()
                 }
             }
         }
+
+        /// <nodoc />
+        public struct Configuration

```

Comment:
>struct [](start = 15, length = 6)

Can we make this a class? That would make Default value a little clearer since default(Configuration) currently does not equal Configuration.Default but is still a "valid" instance.